index.js

```diff
@@ -146,6 +150,31 @@ function rmFate(rowId, channel) {
     );
 }
 
+function rmLastFate(channel) {
+    const params = {
+        icon_emoji: ':fate_wheel_avatar:'
+    }
+    db.all(`SELECT rowid,fateText,epochDateLastUsed 
+                FROM fates 
+                WHERE epochDateLastUsed IS NOT NULL
+                ORDER BY epochDateLastUsed 
+                DESC 
+                LIMIT 10;`, (err, rows) => {
```

Comment:
don't need 10 rows if you only need the most recent one

Reply:
derp, I'd done that but undid it when I got lost in the code and backed out some edits to another function